portal/pom.xml

```diff
@@ -93,7 +93,7 @@
             <configuration>
               <artifactItems>
                 <artifactItem>
-                  <groupId>com.github.adamabeshouse</groupId>
+                  <groupId>com.github.cbioportal</groupId>
                   <artifactId>cbioportal-frontend</artifactId>
                   <version>iviz-SNAPSHOT</version>
```

Comment:
@zhx828 @adamabeshouse Just a note here. Using `iviz-SNAPSHOT` is fine for dev btw, but don't get to this to `rc`. Makes it hard to figure out when errors get introduced, cause compilation differs over time. Also note that people have to clear their local mvn cache to actually get latest iviz branch (mvn doesn't know when to update `iviz-SNAPSHOT`)